resources/sync_config_files.py

```diff
@@ -0,0 +1,705 @@
+# Copyright 2024 CS Group
```

Comment:
Note: run this script with `run poetry run python resources/sync_config_files.py` from the rs-server root directory to create all configuration files from templates and copy them to rs-demo, rs-helm, rs-infrastructures (these projects must be checked out in your same local directory than rs-server)